cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/planner/Dag.java

```diff
@@ -264,7 +215,7 @@ public String removeSource() {
   public List<String> linearize() {
     List<String> linearized = new ArrayList<>();
 
-    Dag copy = new Dag(sources, sinks, outgoingConnections, incomingConnections);
+    Dag copy = new Dag(outgoingConnections, incomingConnections);
```

Comment:
nope, it was true when i first wrote the method but not anymore.  Will correct the javadoc.